index.html

```diff
@@ -4853,6 +4853,66 @@ <h3>Issuer</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h4>Holder</h4>
+        <p>
+The value associated with the <code>holder</code> <a>property</a> is expected
+to identify the <a>holder</a> to the <a>verifier</a>.
+        </p>
+        <p>
+Relevant metadata about the <code>holder</code> <a>property</a> is expected
+to be available to the <a>verifier</a>. For example, a <a>holder</a> can
+publish information containing the verification material used to secure
+<a>verifiable presentations</a>. This metadata is relevant when
+checking proofs on <a>verifiable presentations</a>.
+        </p>
+        <p>
+See the <a data-cite="VC-IMP-GUIDE/#subject-holder-relationships"></a> and
+<a data-cite="VC-USE-CASES#user-tasks"></a> for additional examples related to <a>subject</a> and <a>holder</a>.
+        </p>
+
+        <p class="note">
+`issuer`, `subject`, and `holder` are graph nodes, which support multiple representations, 
+making validating that these roles are performed by the same entity potentially complex.
+        </p>
+        <ul>
+          <li>
+<a href="#issuer">Issuer</a> defines expressions of `issuer` in <a>credentials</a>
+          </li>
+          <li>
+<a href="#presentations-0">Presentations</a> defines expressions of `holder` in <a>presentations</a>
+          </li>
+          <li>
+<a href="#credential-subject">Credential subject</a> defines expressions of `credentialSubject` in <a>credentials</a>
+          </li>
+        </ul>
+        <p>
+A <a>verifier</a> might conjecture that a <a>verifiable presentation</a> <a>holder</a> 
+is the same entity as a <a>verifiable credential</a> <a>subject</a>, when the following conditions apply:
+        </p>
+        <ul>
+          <li>
+The <a>verifiable presentation</a> is secured, 
+using a mechanism the <a>verifier</a> trusts to protect the integrity of <code>vp+ld+json</code>.
+          </li>
+          <li>
+The <a>verifiable presentation</a> includes one or more <a>verifiable credentials</a> that are secured, 
+using a mechanism the <a>verifier</a> trusts to protect the integrity of `vc+ld+json`.
+          </li>
+          <li>
+            <p>
+When the identifiers for `holder` and `subject` are the same.
+            </p>
+          </li>
+          <li>
+            <p>
+When the verification material used to secure a <a>verifiable presentation</a>
+is also present in the claims about the credential <a>subject</a>, either by value or by reference.
```

Comment:
```suggestion
A <a>verifier</a> might speculate that the <a>holder</a> of a <a>verifiable presentation</a> 
is the same entity as the <a>subject</a> of a <a>verifiable credential</a>, based on one or more of the following conditions:
        </p>
        <ul>
          <li>
The <a>verifiable presentation</a> is secured
using a mechanism the <a>verifier</a> trusts to protect the integrity of <code>application/vp+ld+json</code>.
          </li>
          <li>
The <a>verifiable presentation</a> includes one or more <a>verifiable credentials</a> that are secured 
using a mechanism the <a>verifier</a> trusts to protect the integrity of <code>application/vc+ld+json</code>.
          </li>
          <li>
            <p>
The identifiers of the <a>holder</a> of the <a>verifiable presentation</a> and the <a>subject</a> of the <a>verifiable credential</a> are the same.
            </p>
          </li>
          <li>
            <p>
The verification material used to secure the <a>verifiable presentation</a>
is also present in one or more claims about the <a>verifiable credentials</a> <a>subject</a>, either by value or by reference.
```

Reply:
@dlongley since I merged suggestions from you on this section before, mind commenting on this? I think we should take it as is.

Reply:
@OR13 it's fine with me.

Reply:
The one change I recommend... I wish I could easily suggest it, would be in this part of the opening statement in the above change request:

```
is the same entity as the <a>subject</a> of a <a>verifiable credential</a>,
```

Should probably be:

```
is the same entity as a <a>subject</a> of a <a>verifiable credential</a>,
```

Allowing for multiple statements ("the" => "a").

Reply:
I can't commit this its outdated.